docs/03-dev-howtos/01-ab-testing.md

```diff
@@ -265,3 +267,85 @@ Optimizely is relatively expensive - several thousand pounds p/month.
+
+2. Ensure that the object name and the name property are the same - but formatted correctly.  
+eg: object `AudioChangeImagePosition` has the name property `audio-change-image-position`.
+```
+object AudioChangeImagePosition extends Experiment(
+  name = "audio-change-image-position",
+  description = "Test the position of the image on audio pages",
+  owners = Owner.group(SwitchGroup.Journalism),
+  sellByDate = new LocalDate(2018, 8, 3),
+  participationGroup = Perc50
+)
+
+```
+3. Add the experiment to the object property `ActiveExperiments.allExperiments` at the top of the file 
+
+4. Switch the test on for your local environment: http://localhost:9000/dev/switchboard You should be able to find it under `Serverside Tests`
+NB: You should be running in `project dev-build` because you will need it to access the /switchboard screen
+
+5. Incorporate the test in your code as a switch, eg:
+
+```
+@import experiments.{ActiveExperiments, AudioChangeImagePosition}
+
+if(ActiveExperiments.isParticipating(AudioChangeImagePosition)) { 
+    //do variant thing 
+ }
+
+```
+
+NB: If your test suddenly stops working in local, check the switchboard again and make sure your test is still switched on. 
+
+## Checking the test
+
+### Test on CODE:
+
+- deploy branch to CODE
+- switch AB test on for CODE: https://frontend.code.dev-gutools.co.uk/dev/switchboard
+
+### Forcing yourself into the test
+
+Severside AB tests use your session ID run through Fastly's Varnish configuration to assign you to a bucket. To trick it into putting you in the test, add the header `relevant-header-name: variant` to your request for a given page. eg: `X-GU-Experiment-50perc: variant` 
+
+A tool like the Chrome extension Header Hacker can help you to do this. You'll need to do this for the different urls you see the Guardian on: eg `localhost`, `https://code.dev-theguardian.com`. Ask for help configuring this if you need it. 
+
+Example Header Hacker configuration for putting yourself into a 50% test on CODE. 
+
+	```
+    Custom Request Headers 
+	ServerTest3	X-GU-Experiment-50perc	Replace With	variant
+    
+    Permanent Header Switches 
+    https://code.dev-theguardian.com/	ServerTest3 (Replace X-GU-Experiment-50perc)
+    
+    ```
+    
+ ## Getting the results 
+ 
+Because it is a server-side test, different html will be rendered. 
+
+Give each component a different `data-component` attribute to distinguish them. This will show up in the `rendered components` list on the page view table. 
+
+You can check this in the Network tab of the page, by filtering for requests sent off to `ophan` and checking what's in the `renderedComponents` property. 
+
+In the data lake search for the presence of that rendered component then correlate that with the event you want to measure eg page ready, clicked play. Ophan events such as these can be added as per the clientside tests above. 
```

Comment:
Maybe link to the [relevant docs](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/19-tracking-components-in-the-data-lake.md#rendered-components) on how to query components in the Data Lake

Reply:
Can do